pkg/output/output.go

```diff
@@ -303,3 +307,65 @@ func (output ContainersOutput) Print(format string) {
 		klog.Errorf("Output format is not supported. The supported formats are json and table only")
 	}
 }
+
+// CombinedOutputFormat has both helm releases and containers info in a backwards compatible way
+type CombinedOutputFormat struct {
+	Helm       []ReleaseOutput `json:"helm"`
+	IncludeAll bool            `json:"include_all"`
+	Container  struct {
+		ContainersOutput
+	} `json:"container"`
+}
+
+// NewHelmAndContainersOutput creates a new HelmAndContainersOutput object ready to be printed
+func NewHelmAndContainersOutput(helm Output, container ContainersOutput) *HelmAndContainersOutput {
+	return &HelmAndContainersOutput{
+		Helm:      helm,
+		Container: container,
+	}
+}
+
+// Print prints the HelmAndContainersOutput to STDOUT
+func (output HelmAndContainersOutput) Print(format string, wide, showOld bool) {
+	switch format {
+	case TableFormat:
+		output.Helm.Print(format, wide, showOld)
+		fmt.Println("")
+		output.Container.Print(format)
+	case JSONFormat:
+		outputFormat := CombinedOutputFormat{
+			Helm:       output.Helm.HelmReleases,
+			Container:  struct{ ContainersOutput }{ContainersOutput: output.Container},
+			IncludeAll: output.Helm.IncludeAll,
+		}
+		data, _ := json.Marshal(outputFormat)
+		fmt.Fprintln(os.Stdout, string(data))
+	}
+}
+
+// Print prints the HelmAndContainersOutput to STDOUT
+func (output HelmAndContainersOutput) ToFile(filename string) error {
+	output.Helm.dedupe()
+	extension := path.Ext(filename)
+	switch extension {
+	case ".json":
+		outputFormat := CombinedOutputFormat{
+			Helm:       output.Helm.HelmReleases,
+			Container:  struct{ ContainersOutput }{ContainersOutput: output.Container},
+			IncludeAll: output.Helm.IncludeAll,
+		}
+		data, err := json.Marshal(outputFormat)
+		if err != nil {
+			klog.Errorf("Error marshaling json: %v", err)
+			return err
+		}
+		err = ioutil.WriteFile(filename, data, 0644)
+		if err != nil {
+			klog.Errorf("Error writing to file %s: %v", filename, err)
+		}
+	default:
+		// TODO - when both flags are used should it have CSV output?!
```

Comment:
@rbren - WDYT about this one?

Reply:
Hmm - yeah I don't think we could do CSV in that case